schemas/app-sre/integration-1.yml

```diff
@@ -55,6 +55,24 @@ properties:
         type: boolean
       early_exit:
         type: boolean
+      shardSpecOverride:
+        type: array
+        items:
+          type: object
+          additionalProperties: false
+          properties:
+            shardingStrategy:
+              type: string
+              enum:
+                - per-aws-account
+            awsAccount:
+              "$ref": "/common-1.json#/definitions/crossref"
+              "$schemaRef": "/aws/account-1.yml"
+            imageRef:
+              type: string
+          required:
+          - awsAccount
+          - imageRef
```

Comment:
if this is now a duplicate, i would suggest extracting it to a separate schema

Reply:
Good point! Will do

Reply:
I refactored the schema. However this now crashes the qontract-server. Validation works fine though.

Reply:
Nevermind, was related to local image mess